airbyte-webapp/src/pages/connections/ConnectionPage/ConnectionPage.tsx

```diff
@@ -37,13 +38,22 @@ export const ConnectionPage: React.FC = () => {
   const { connectionId = "" } = useParams<{
     connectionId: string;
   }>();
+  const location = useLocation();
+  const isReplicationPage = useMemo(
+    () => location.pathname.includes(`/${ConnectionRoutePaths.Replication}`),
+    [location.pathname]
+  );
 
   useTrackPage(PageTrackingCodes.CONNECTIONS_ITEM);
 
   return (
     <ConnectionEditServiceProvider connectionId={connectionId}>
       <ResourceNotFoundErrorBoundary errorComponent={<StartOverErrorView />}>
-        <MainPageWithScroll headTitle={<ConnectionHeadTitle />} pageTitle={<ConnectionPageTitle />}>
+        <MainPageWithScroll
+          headTitle={<ConnectionHeadTitle />}
+          pageTitle={<ConnectionPageTitle />}
+          noBottomPadding={isReplicationPage}
+        >
```

Comment:
nitpick: would be a bit more declarative and cleaner (IMO) to put `<MainPageWithScroll>` inside each route, then just set `noBottomPadding={true}` in the `ConnectionRoutePaths.Replication` route. Then all the styling logic for that route is in one place, and we don't have to do a potentially brittle check based on the current URL.